docs/source/reference/respawn/player.rst

```diff
@@ -11,60 +11,75 @@ Functions for getting player, and methods of the player object
 
 .. cpp:function:: array<entity> GetPlayerArray()
 
-
 .. cpp:function:: array<entity> GetPlayerArrayOfTeam( int team )
 
+.. cpp:function:: array<entity> GetPlayerArrayOfTeam_Alive( int team )
+
 .. cpp:function:: array<entity> GetPlayerArrayOfEnemies_Alive( int team )
 
-.. cpp:class:: player : public entity	
+.. cpp:function:: array<entity> GetSortedPlayers( IntFromEntityCompare compareFunc, int team)
+
+    returns list of every player in ``team`` sorted by ``compareFunc``. If ``team`` is 0 returns a sorted array of every player.
+    `squirrel compare function example <http://www.squirrel-lang.org/squirreldoc/reference/language/builtin_functions.html#array.sort>`_
+    .. code-block:: javascript
+        GetSortedPlayers(function(entity player1, entity player2) {
```

Comment:
The automation messed up and didnt show, but this is the temporary build for this PR:

https://r2northstar--3.org.readthedocs.build/en/3/reference/respawn/player.html

take a look at the codeblock in the example, its kinda messed up. This is because in ReStructuredText, something following a directive like codeblock without a empty line, is considered a attribute of that directive.

Basically, you need to add empty lines before and after directives.

Other than that, this is a great PR and exactly the type of thing we need.